gene/mutate.go

```diff
@@ -231,16 +266,18 @@ func Insertion(g *Gene, genomePos int, alt []dna.Base) (EffectPrediction, error)
 		for i := 0; i < len(alt); i++ {
 			g.featureArray[genomeIndexPos+1+i] = fillVal
 		}
-		//TODO: Effect Prediction WIP
-		//var endCdnaOffset int
-		//answer.CdnaPos, answer.CdnaDist, err = GenomicPosToCdna(g, genomePos + 1)
-		//_, endCdnaOffset, err = GenomicPosToCdna(g, genomePos + 1 + (len(alt) - 1))
-		//if numbers.AbsInt(endCdnaOffset) < numbers.AbsInt(answer.CdnaDist) {
-		//	answer.Consequence = checkSplice(endCdnaOffset)
-		//} else {
-		//	answer.Consequence = checkSplice(answer.CdnaDist)
-		//}
+
+		var endCdnaOffset int
+		answer.CdnaPos, answer.CdnaDist, err = GenomicPosToCdna(g, genomePos+1)
+		_, endCdnaOffset, err = GenomicPosToCdna(g, genomePos+1+(len(alt)-1))
```

Comment:
I wrote most of this before I went home for the holidays so I am pretty lost too. I have no clue why I kept adding 1 to the genomePos when calling GenomicPosToCdna. This will be fixed.